src/Scope.php

```diff
@@ -412,8 +412,10 @@ protected function fireTransformer($transformer, $data)
             $transformedData = $this->manager->getSerializer()->mergeIncludes($transformedData, $includedData);
         }
 
-        //Stick only with requested fields
-        $transformedData = $this->filterFieldsets($transformedData);
+        // Stick only with requested fields if we have more than one
+        if (is_array($transformedData)) {
+            $transformedData = $this->filterFieldsets($transformedData);
+        }
```

Comment:
should we throw an exception here? or is it thrown somewhere above?

Reply:
The hope is the scalar values will flow out if they are returned by the transformer so that you can achieve something like `['data' => [1, 2, 3]]`, I'm realizing now that this isn't going to work since we need to return `array` in all cases anyway.